toolkit/tools/pkg/imagecustomizerlib/partitionutils.go

```diff
@@ -30,25 +30,25 @@ var (
 	partitionNumberRegex = regexp.MustCompile(`^/dev/loop\d+p(\d+)$`)
 )
 
-func findPartitions(buildDir string, diskDevice string) ([]*safechroot.MountPoint, error) {
+func findPartitions(buildDir string, diskDevice string) ([]*safechroot.MountPoint, []diskutils.PartitionInfo, error) {
 	var err error
 
 	diskPartitions, err := diskutils.GetDiskPartitions(diskDevice)
```

Comment:
nit: Move the call to `GetDiskPartitions` from the `findPartitions` function into the `connectToExistingImageHelper` function. And then pass the `diskPartitions` value into `findPartitions` as a param. IMHO, this will make the code slightly cleaner.